sp800-63a/sec6_dc.md

```diff
@@ -34,25 +36,27 @@ This section applies when the secondary authenticator is directly tied to the au
 2. The CSP SHOULD check the status of the original authenticator daily.
-3. The CSP SHALL obtain and verify a copy of a biometric recorded when the primary authenticator was issued. An example of such a biometric is the signed biometric data object, however if the biometric reference is not available from the original AAL 3 authenticator, it may be obtained elsewhere, as long as its authenticity is assured.
-4. The CSP SHALL compare a fresh biometric sample obtained in person from the applicant to the reference biometric retained from the original, primary AAL 3 authenticator and determine that they match.
-5. The CSP SHALL determine that the authenticator of the original, primary authenticator meets all requirements of an AAL 3 authenticator.
+3. The CSP SHALL obtain and verify a copy of a biometric recorded when the primary authenticator was issued. An example of such a biometric is the signed biometric data object, however if the biometric reference is not available from the original authenticator, it may be obtained elsewhere, as long as its authenticity is assured. **MG: How do we know it's an AAL 3 authenticator?**
+4. The CSP SHALL compare a fresh biometric sample to the reference biometric obtained from the original, primary authenticator and determine that they match.
+5. The CSP SHALL determine that the derived authenticator meets all the requirements of the primary authenticator's AAL.
+
+>**MG: Leaving this but I think we can simplify.**
 
 ## <a name="prior"></a>6.2 Issuance Requirements for Accepting Results of Prior Proofing
 
-Where the applicant already possesses an authenticator from a trusted CSP, the new CSP could choose to "inherit" the successful identity proofing transaction performed in a prior encounter at the original CSP, by verifying possession and control of the authenticator associated with the original CSP.  The following details the requirements for the new CSP:
+Where the applicant already possesses an authenticator from a trusted CSP, the new CSP could choose to inherit the identity proofing performed in a prior encounter at the original CSP, by verifying possession and control of the authenticator associated with the original CSP.  The following details the requirements for the new CSP:
 
 ### 6.2.1. General Requirements
 
-Unless otherwise specified, the term "CSP" referenced below pertains to the new CSP that depends on the identity proofing performed by the original CSP.
+Unless otherwise specified, the term CSP in this section pertains to the new CSP that depends on the identity proofing performed by the original CSP.
 
 1. Before issuance, the CSP SHALL verify that the corresponding authenticator is possessed and controlled by the claimant.
 2. Before issuance, the CSP SHOULD verify the original authenticator status. The CSP SHALL NOT issue a derived authenticator if status indicates any type of termination, disablement, revocation, or expiration.
```

Comment:
Verifying the status of the credential, I think. Really we want to make sure that the subject can authenticate to the original CSP at a given AAL. We don't need to be so concerned with the authenticators themselves.